ui/src/app/components/help-sidebar.tsx

```diff
@@ -390,28 +396,47 @@ export const HelpSidebar = fp.flow(
             ),
             showFooter: true,
           };
-        case 'runtime':
+        case 'apps':
           return {
             headerPadding: '0.75rem',
-            renderHeader: () => (
-              <div>
-                <h3
-                  style={{
-                    ...styles.sectionTitle,
-                    lineHeight: 1.75,
-                  }}
-                >
-                  Cloud analysis environment
-                </h3>
-              </div>
-            ),
-            bodyWidthRem: '30',
+            renderHeader:
```

Comment:
We you click the settings button for the Jupyter application, it brings you to the "Cloud analysis environment" button. It does not look like there is a way to get back to the apps page from here. Should there be a back button?
![image](https://user-images.githubusercontent.com/24514630/202501176-8e552157-facc-4a9d-a44f-391c282118fd.png)